### PR TITLE
Release/322.0.0

### PR DESCRIPTION
**Pull request by imblue-dabadee**
## Explanation

This release fixes an edge case `PhishingController` where empty phishing lists could trigger API requests with invalid `-Infinity` timestamps. It also exports `RecommendedAction` so that clients that use `scanUrl` can correctly import it.

### `@metamask/phishing-controller`

- **FIXED**:  Fixes an edge case `PhishingController` where empty phishing lists could trigger API requests with invalid `-Infinity` timestamps
- **FIXED: Correctly adds RecommendedAction**

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate
- [x] I've prepared draft pull requests for clients and consumer packages to resolve any breaking changes

### CODE REVIEW

File: packages/phishing-controller/CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.4.1]
+
 ### Fixed
 
 - Fixed an edge case in `PhishingController` where empty phishing lists could trigger API requests with invalid `-Infinity` timestamps ([#5385](https://github.com/MetaMask/core/pull/5385))
+- Fixed `RecommendedAction` not being exported correctly ([#5456](https://github.com/MetaMask/core.git/pull/5456))
```
Comment (cryptotavares) [inline packages/phishing-controller/CHANGELOG.md:15]:
```suggestion
- Fixed `RecommendedAction` not being exported correctly ([#5456](https://github.com/MetaMask/core/pull/5456))
```

**Review by cryptotavares — APPROVED** [review]
